.github/workflows/amd64_macos_bazel.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   # Building using the github runner environement directly.
   bazel:
-    runs-on: macos-latest
+    runs-on: macos-13
```

Comment:
Can you add `# Using x86 processors, ref: https://github.com/actions/runner-images` after `macos-13` ?